lang/py3/avro/io.py

```diff
@@ -67,16 +68,33 @@
 STRUCT_CRC32 = struct.Struct('>I')   # big-endian unsigned int
 
 
+class TermColors:
+  BLUE = '\033[94m'
+  GREEN = '\033[92m'
+  RED = '\033[91m'
+  ENDC = '\033[0m'
```

Comment:
Is there any way we can disable these term colors when stdout is not a tty, or when the termcap doesn't support it?

Reply:
Absolutely, I think it's enough to check `sys.stdout.isatty()`. I'll do it.

Reply:
@kojiromike I considered using `colorama`, but finally I decided not to add another dependency and just check if the terminal is tty.

Reply:
Have you considered using an `extras_require` on colorama?

Reply:
That would be good idea, but the purpose of `colorama` would be to clean the stdout from all color strings, so the code could work for ex. on Windows. Then the lib should be mandatory requirement, otherwise Windows users would get some weird strings in output. 

Another problem I discovered is that `colorama` makes the code untestable, because the lib injects different ANSI escape character sequences depending on the OS. I experimented with this solution and found that there is no easy way to unit-test such code.